network/wsNetwork.go

```diff
@@ -764,7 +765,8 @@ func (wn *WebsocketNetwork) setup() {
 		SupportedProtocolVersions = []string{wn.config.NetworkProtocolVersion}
 	}
 
-	wn.messagesOfInterestCond = sync.NewCond(&wn.messagesOfInterestMu)
+	wn.messagesOfInterestNotify = make(chan uint32, 2)
```

Comment:
why channel of size 2?

Reply:
both of the consumers of the notification go back to the source to get the latest value and write out a result idempotently; but they need to be notified if things might have changed _again_ since they started handling the last one.

Reply:
for messagesOfInterestNotify though, you will always continue to get notifications each round, so why not just skip the round you missed rather than get notified that you missed (at most) one round?

Reply:
Sorry, meant messagesOfInterestRefresh, it seems like you only need size of 1 for the per-round notifications. For the other Notify channel I think I understand the reasoning about missing an update, but it seems like you already know if you have changed the MOI every time you handle a block notification / refresh based on whether you called Register or Deregister.